lib/teiserver/account.ex

```diff
@@ -1030,6 +1030,61 @@ defmodule Teiserver.Account do
     Repo.delete(rating)
   end
 
+  @doc """
+  Returns the player's rank when compared to other players by rating.
+
```

Comment:
In teiserver "rank" means their chevron level indexed at 0. So we have to be careful here since you are using the term "rank" in a different way. Perhaps add a comment that this is ranking relative to other players completely unrelated to chevron levels. In teiserver db, the rank of a user relates to chev levels.